dev/src/lobster/parser.h

```diff
@@ -584,6 +584,7 @@ struct Parser {
             case T_STRTYPE:   dest = type_string;     lex.Next(); break;
             case T_COROUTINE: dest = type_coroutine;  lex.Next(); break;
             case T_RESOURCE:  dest = type_resource;   lex.Next(); break;
+			case T_ANYTYPE:   dest = type_any;        lex.Next(); break;
```

Comment:
I'm doing by the third comment, but what is the significance of changing `any` to `?`? I mean behavior with anytype token should be same as with a wildcard one, since it'd be looked ahead explicitly. Or do you just think <?> is better looking?

Reply:
I guess we currently don't have `any` as a type anymore you can specify, so I am trying to not introduce it here. Besides, it is even confusing, since `any` isn't a supertype of `int` for example. We're purely using `any` here as an escape hatch we can test for. Maybe we should be using `type_undefined` instead to be even clearer.